Formula/ipv6calc.rb

```diff
@@ -15,7 +19,52 @@ class Ipv6calc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "748772297ce211e30da123ccab4c9204738703e62cfe1ab90238dac434d9fb70"
   end
 
+  uses_from_macos "perl"
+
+  resource "URI::Escape" do
+    on_linux do
+      url "https://cpan.metacpan.org/authors/id/E/ET/ETHER/URI-1.72.tar.gz"
+      sha256 "35f14431d4b300de4be1163b0b5332de2d7fbda4f05ff1ed198a8e9330d40a32"
+    end
+  end
+
+  resource "HTML::Entities" do
+    on_linux do
+      url "https://cpan.metacpan.org/authors/id/O/OA/OALDERS/HTML-Parser-3.76.tar.gz"
+      sha256 "64d9e2eb2b420f1492da01ec0e6976363245b4be9290f03f10b7d2cb63fa2f61"
+    end
+  end
+
+  resource "DIGEST::Sha1" do
+    on_linux do
+      url "https://cpan.metacpan.org/authors/id/G/GA/GAAS/Digest-SHA1-2.13.tar.gz"
+      sha256 "68c1dac2187421f0eb7abf71452a06f190181b8fc4b28ededf5b90296fb943cc"
+    end
+  end
```

Comment:
Can we not put `resource` blocks in `on_linux` blocks, so that we only have one `on_linux` block instead of three?